docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,377 @@
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant with terraza in front of the lake",
+    "menu": null,
+    "locale": "en",
+    "published_at": "2021-04-07T12:51:43.681Z",
+    "created_at": "2021-04-07T12:51:43.695Z",
+    "updated_at": "2021-04-07T12:51:43.695Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+}
+```
+
+:::
+::::
+
+To create a localized entry for a locale different from the default one, add the `locale` attribute to the body of the POST request:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants
+```
+
+```json
+{
+  "name": "Restaurant F",
+  "slug": "restaurant-f",
+  "stars": 1,
+  "description": "Restaurant face à la mer",
+  "locale": "fr"
+}
+```
+
+:::
+
+::: tab Response
+**Example response**
+
+```json
+{
+    "id": 8,
+    "name": "Restaurant F",
+    "slug": "restaurant-f",
+    "price_range": null,
+    "contact_email": null,
+    "stars": 1,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant face à la mer",
+    "menu": null,
+    "locale": "fr",
+    "published_at": "2021-04-07T13:22:46.589Z",
+    "created_at": "2021-04-07T13:22:46.617Z",
+    "updated_at": "2021-04-07T13:22:46.617Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+}
+```
+
+:::
+::::
+
+## Creating a localization for an existing entry
+
+To create another localization for an existing localized entry, send a POST request to the appropriate URL depending on the type of content:
+
+| Content-Type    | Request URL format                       |
+|:----------------|:-----------------------------------------|
+| Single Type     | `POST /{content-type}/:id/localizations` |  
+| Collection Type | `POST /{content-type}/localizations`     |
+
+
+::: warning CAUTION
+When creating a localization for existing localized entries, the body of the POST request can only accept localized fields.
+:::
+
+When sending the request to a single type, Strapi will:
+
+1. use the `id` as base entry for the non-localized fields and copy them in the new entry
+2. then create a new entry for the given locale and link it with the base entry.
+:::: tabs
+
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants/8/localizations
+```
+
+**body**
+<!-- * We never used "body" before, see existing entries in Content API. I think it would be a good idea — so it's even more explicit that this code block is the body of the request. If we agree on this, I should update all existing examples in Content API to follow this format. -->
```

Comment:
Fine by me, but maybe you can do it in another PR so we don't make this one bigger than it needs to be?

Reply:
Sure!